src/main/resources/assets/js/components/AppHealthComponent.jsx

```diff
@@ -0,0 +1,78 @@
+/** @jsx React.DOM */
+
+var React = require("react/addons");
+
+var AppHealthComponent = React.createClass({
+  displayName: "AppHealthComponent",
+
+  getHealthData: function () {
+    var model = this.props.model;
+
+    var tasksWithUnknownHealth = Math.max(
+      model.get("tasksRunning") -
+      model.get("tasksHealthy") -
+      model.get("tasksUnhealthy"),
+      0
+    );
+
+    var healthData = [
+      {quantity: model.get("tasksHealthy"), className: "health-bar-healthy"},
+      {quantity: model.get("tasksUnhealthy"), className: "health-bar-unhealthy"}, /* jscs:disable maximumLineLength */
```

Comment:
Please don't break the 80 character rule on purpose. It is only for when React html is not behaving (because the transformer is not always making the best choices)